src/elements/content-sidebar/activity-feed/comment/BaseComment.js

```diff
@@ -446,7 +451,7 @@ export const Replies = ({
                                     mentionSelectorContacts={mentionSelectorContacts}
                                     onCommentEdit={onCommentEdit}
                                     onSelect={onReplySelect}
-                                    onDelete={noop}
```

Comment:
I don't see any codepaths that were particular to this being `noop`---which surprising since `onEdit` has some `noop`-specific paths, like  `const canEdit = onEdit !== noop && (permissions.can_edit ?? false);`. Mentioning it juuust in case.